TreeDB/documentservice/service.go

```
@@ -446,6 +446,20 @@ func (s *Service) CountDocuments(ctx context.Context, index string, req CountDoc
 	if err != nil {
 		return CountDocumentsResponse{}, err
 	}
+	if req.Filter == nil {
+		count := 0
+		_, err = col.ScanDocumentIDsFunc(maxServiceScanDocuments, func([]byte) (bool, error) {
```

golangci-lint (2.12.2)

[error] Function `ScanDocumentIDsFunc->flushBufferedWrites->flushBufferedWritesWithRawPublishStateAndCoverage->flushBufferedWritesLockedWithRawPublishState->flushBufferedIndexedLockedWithRawPublishState->flushBufferedIndexedLockedWithRawPublishState$5->publishBufferedOrderedRootDeltaBatchGroupWithCommandWAL->PublishStagedOrderedRootDeltaBatchGroupWithCommandWALAndSystemDeltaBuilder->publishOrderedRootDeltaBatchGroupWithSystemDeltaBuilderWithOptions->publishOrderedRootDeltaBatchGroupWithSystemDeltaBuilderSerialized->finalizeOrderedRootPublishWithCommandWALOptions->finalizeCommitReleasingRootSerialization->finalizeCommitLockedWithOptions->finalizeQueuedRootPublicationV1->captureDurableRootResourcesFromBaseV1->captureDurableValueLogResourcesV1->scanCandidateValueLogReferencesV1->scanCandidateExternalReferencesV1->scanCandidateExternalReferencesV1$1` should pass the context parameter

(contextcheck)